frontend/src/components/eMIB/EditTask.jsx

```diff
@@ -115,24 +96,8 @@ class EditTask extends Component {
     this.props.onChange({ ...this.state, reasonsForAction: newReasonForAction });
   };
 
-  onTaskTooltipFocus = () => {
-    this.setState({ taskTooltipIcon: "fas fa-question-circle" });
-  };
-
-  onTaskTooltipBlur = () => {
-    this.setState({ taskTooltipIcon: "far fa-question-circle" });
-  };
-
-  onReasonsForActionTooltipFocus = () => {
-    this.setState({ reasonsForActionTooltipIcon: "fas fa-question-circle" });
-  };
-
-  onReasonsForActionTooltipBlur = () => {
-    this.setState({ reasonsForActionTooltipIcon: "far fa-question-circle" });
-  };
-
```

Comment:
If we figure out how to retrieve our previous icon shape (question mark with a circle), we'll need to put that logic back in order to change the icon color depending on if the tool tip is on or not.